### ISSUE TITLE
Wrong augmentation parameter checking  for tinyimagenet

### ISSUE THREAD

**Issue by WeiWeic6222848** [uitrbn/IDM#3]
The following line seems to be off, it looks like that if the aug parameter is true, the raw image is retrieved, while if it is set to false, the downscaling is done, is this intended?
https://github.com/uitrbn/IDM/blob/fe23bbcd26f664f7f27479e298866a2ff1cc5005/IDM_tiny.py#L135-L143

Compared to the same lines in cifar10 as example, where the if clause is reversed.
https://github.com/uitrbn/IDM/blob/fe23bbcd26f664f7f27479e298866a2ff1cc5005/IDM_cifar10.py#L138-L145

**Comment by uitrbn** [comment]
No, it is a bug. Thank you for pointing it out. We will fix it later.
